src/scripts/tweaks.json

```
@@ -13,6 +13,11 @@
       "label": "Attempt to revert the post footer redesign",
       "default": false
     },
+    "hide_note_count_types": {
+      "type": "checkbox",
+      "label": "Hide the type indicators next to the note count",
```

not sure "indicators" is the best descriptor here

```suggestion
      "label": "Hide the note type badges next to the note count",
```

---

👍